### PR TITLE
Store and retrieve daily clone statistics from Redis db

### PR DESCRIPTION
**Pull request by sschneid**

### CODE REVIEW

File: lib/vmpooler/api.rb [hunk 1]
```diff
@@ -189,23 +188,17 @@ def hostname_shorten hostname
...
             result['status'] = 0
           end
 
-          if ( clone_times.any? )
-            result['clone_average'] = clone_times.map( &:to_f ).reduce( :+ ) / clone_times.size
-          end
-
           result['capacity_perecent'] = ( result['capacity_current'].to_f / result['capacity_total'].to_f ) * 100.0
 
+          result['clone_total'] = $redis.hlen('vmpooler__clone__'+Date.today.to_s)
```
Comment (colinPL) [inline lib/vmpooler/api.rb]: This is total for today, not for 24 hours. I do not see a piece that calculates `now - 24 hours`.

Comment (sschneid) [inline lib/vmpooler/api.rb]: Nobody said in this PR it was for 24-hours :)

File: lib/vmpooler/api.rb [hunk 2]
```diff
@@ -189,23 +188,17 @@ def hostname_shorten hostname
...
+          result['clone_average'] = $redis.hvals('vmpooler__clone__'+Date.today.to_s).map( &:to_f ).reduce( :+ ) / result['clone_total']
```
Comment (colinPL) [inline lib/vmpooler/api.rb]: Should this have any guards incase `result['clone_total']` is `0`?

Comment (sschneid) [inline lib/vmpooler/api.rb]: Patched

**Pull request merged**